az-snp-vtpm/README.md

```diff
@@ -25,3 +25,45 @@ Retrieve SEV-SNP report, validate and print it:
 ```bash
 sudo ./snp-vtpm -p
 ```
+
+## Example Project
+
+There is a project in the `./example` folder depicting how the crate can be leveraged in a Remote Attestation flow. **Note:** the code is merely illustrative and doesn't feature exhaustive validation, which would be required in a production scenario.
+
+```bash
+cargo b -p example
+```
+
+## SEV-SNP Report & vTPM 
+
+The vTPM is linked to the SEV-SNP report via the vTPM Attestation Key (AK). The public AK is part of a Runtime Data struct, which is hashed and submitted as Report Data when generating the SNP report. To provide freshness guarantees in an attestation exchange we can request a vTPM quote with a nonce. The resulting message is signed by the AK.
+
+```
+                              ┌────────────────────────┐
+                              │ HCL Data               │
+                              │                        │
+                              │ ┌──────────────────────┴─┐  ─┐
+                              │ │ Runtime Data           │   │
+                              │ │                        │   │
+    ┌──────────────────────┐  │ │ ┌────────────────────┐ │   ├─┐
+  ┌─┤ vvTPM AK             ├──┼─┼─┤ vTPM Public AK     │ │   │ │
```

Comment:
```suggestion
  ┌─┤ vTPM Public AK       ├──┼─┼─┤ vTPM Public AK     │ │   │ │
```

Reply:
the private part of the AK is signing the msg